.github/workflows/hail_update.yaml

```diff
@@ -52,7 +55,7 @@ jobs:
 
     - name: "deploy server-test"
       run: |
-        gcloud run deploy server-test --region australia-southeast1 --no-allow-unauthenticated --platform managed --set-env-vars=DRIVER_IMAGE=$DRIVER_IMAGE --image $SERVER_IMAGE
+        gcloud run deploy server-test --region australia-southeast1 --no-allow-unauthenticated --platform managed --set-env-vars=DRIVER_IMAGE=$DRIVER_IMAGE,TZ=Australia/Sydney,HAIL_SHA=${{ github.event.inputs.hail_sha }} --image $SERVER_IMAGE
```

Comment:
TZ is new, was there something else that meant we had to add that?

Reply:
This is just for consistency with the prod version.

Reply:
I don't think `TZ` is used anywhere, so I'll remove this in a follow-up PR.